### PR TITLE
Stage2

### LINKED ISSUE #11: Description in readme of Stage2

### PR DESCRIPTION
**Pull request by tyczynoprzemyslaw**
If you can check everything, especially tests, i have strong feeling it's should be more. If you can give me name of tests I'll try to implement them.

### CODE REVIEW

**Review by PawelDabrowski83 — APPROVED** [review]
w statycznej analizie kodu nie widzę uwag, także wrzucam na repo i zobaczymy jak zadziała :+1:

File: src/main/java/com/facebook/bitEncoderDecoder/Stage1.java [hunk 1]
```diff
@@ -4,11 +4,6 @@
 
     public String encode(String input) {
         StringBuilder encodeString = new StringBuilder();
```
Comment (PawelDabrowski83) [inline src/main/java/com/facebook/bitEncoderDecoder/Stage1.java]: It seems that you are trying to delete some of Stage1 code in this pull. I will try to stop you

Comment (PawelDabrowski83) [reply src/main/java/com/facebook/bitEncoderDecoder/Stage1.java]: ok, I don't know how to do it

**Review by PawelDabrowski83 — CHANGES_REQUESTED** [review]
please undo changes in Stage1

**Comment by tyczynoprzemyslaw** [comment]
I will make another pull request, i close this one.

**Pull request closed without merging**